Source/Core/DiscIO/VolumeDirectory.cpp

```diff
@@ -353,9 +353,10 @@ void CVolumeDirectory::BuildFST()
   m_fst_data.clear();
 
   File::FSTEntry rootEntry = File::ScanDirectoryTree(m_root_directory, true);
-  u32 name_table_size = ComputeNameSize(rootEntry);
+  u32 name_table_size = Common::AlignUp(ComputeNameSize(rootEntry), 1ull << m_address_shift);
```

Comment:
It would be nice if you also could change that `_dbg_assert_` to `_assert_` while you're at it. The only reason to use the debug version of the assert macro is to improve the performance in release builds, but using it in code that gets called as rarely as this won't affect anything in a measurable way.